medusa/search/manual.py

```diff
@@ -276,7 +276,8 @@ def get_provider_cache_results(indexer, show_all_results=None, perform_search=No
         cached_results = [dict(row) for row in sql_total]
         for i in cached_results:
             i['quality_name'] = Quality.split_quality(int(i['quality']))
-            i['time'] = datetime.fromtimestamp(i['time']).strftime(app.DATE_PRESET + ' ' + app.TIME_PRESET)
+            i['time'] = datetime.fromtimestamp(i['time'])
+            i['time_pretty'] = pretty_date(i['time'])
```

Comment:
Yes no point in losing accuracy, the universal time format can still be show on a title tooltip or even a qTip tooltip.